src/main/kotlin/org/rust/ide/inspections/fixes/RemoveTypeParameter.kt

```diff
@@ -0,0 +1,30 @@
+package org.rust.ide.inspections.fixes
+
+import com.intellij.codeInspection.LocalQuickFix
+import com.intellij.codeInspection.ProblemDescriptor
+import com.intellij.openapi.project.Project
+import org.rust.lang.core.psi.RsBaseType
+import org.rust.lang.core.psi.RsCallExpr
+import org.rust.lang.core.psi.RsMethodCallExpr
+import org.rust.lang.core.psi.RsPathExpr
+import org.rust.lang.core.psi.ext.RsCompositeElement
+
+class RemoveTypeParameter(val element: RsCompositeElement) : LocalQuickFix {
```

Comment:
All fix classes in the plugin have `Fix` suffix. Let's stick to this naming convention.